app/config/default.json

```diff
@@ -18,6 +18,7 @@
     "logLevel": "http",
     "maxAttempts": "3",
     "port": "3000",
-    "smtpHost": "apps.smtp.gov.bc.ca"
+    "smtpHost": "apps.smtp.gov.bc.ca",
+    "blockDoNotReplySender": false
```

Comment:
It's generally better to test for this by having this be undefined - if it is defined/true, then block; else if undefined, don't. Environment variables do not have a state of "false", as a string "false" could sometimes be misinterpreted as a true value.